src/main/webapp/app/overview/exercise-details/course-exercise-details.component.ts

```diff
@@ -65,10 +68,14 @@ export class CourseExerciseDetailsComponent implements OnInit, OnDestroy {
     }
 
     loadExercise() {
+        const options = {};
```

Comment:
I think this logic should not be part of the components, but of the services.
It would be the same for different components that call the same methods & also it is not nice that every component now has to import the localStorageService (this e.g. affects our tests).

Also this should probably be part of a util to be reusable.